src/lib.rs

```diff
@@ -136,20 +135,24 @@ impl manager::NotificationWatcher for ZWaveWatcher {
                 let controller = notification.get_controller();
                 let mut state = self.get_state();
                 if !state.controllers.contains(&controller) {
-                    println!("Found new controller: {:?}", controller);
+                    println!("Found new controller: {}", controller);
                     state.controllers.insert(controller);
                 }
             },
+            NotificationType::Type_NodeNew => {
+                let node = notification.get_node();
+                println!("NodeNew: {}", node);
```

Comment:
I don't know how this will be useful to the foxbox yet. Basically this means there is a new node in the network, that wasn't configured in the .xml file. Could it be useful to the foxbox to ask the user to configure it ?